docs/design/auditing-containment.md

```diff
@@ -0,0 +1,168 @@
+# Auditing and Node Containment
+
+## Abstract
+
+This design doc outlines how we will implement "containment mode," in which the auditing service monitors a node that has accepted an audit request, but refuses to send the requested data.
+Currently in the codebase, nodes that do this will be marked as having audit failures.
+However, we don't want to mark nodes as offline or immediately mark an audit failure if this specific case arises.
+Instead, we want to "contain" these nodes by retrying the original audit until they either eventually respond and pass the audit, or refuse to respond a certain number of times and ultimately receive an audit failure mark.
+
+## Main Objectives
+1. Nodes won’t be able to “escape” audits for specific data by refusing to send requested data.
+2. The audit service should not audit contained nodes for other data, and should verify their originally requested data.
+3. The overlay service should not select contained nodes.
+4. A ContainmentDB holds metadata required to verify contained nodes' originally requested data.
+
+## Background
+
+The whitepaper section 4.13 talks about containment mode as follows:
+```
```

Comment:
Can you change the format of this paragraph to use `>`?

Otherwise when rendered all the test gets in one line with a horizontal scroll

Example

> Given a specific storage node, an audit might reveal that ...